### DISCUSSION TITLE
Signac equivalent in Python

### DISCUSSION THREAD

**Discussion by Mayank0512** [stuart-lab/signac#880] [General]
I am trying to find the equivalent package of Signac in Python. Can anyone please help me out here ?
I found that bio-conductor uses rpy2 to bridge the gap between Python and R .Is something similar possible with Signac too?
Thank you for your help and time
Regards,
Mayank Ghogale

**Comment by timoast** [comment]
Please see https://github.com/timoast/signac/issues/879

(there's no need to open both issue and discussion for the same question)

**Comment by Mayank0512** [comment]
Okay yes sorry about that
Thank you for your help
